### PR TITLE
List orders by date and toPayment()

### PR DESCRIPTION
**Pull request by marcushanses**

### CODE REVIEW

File: docs/merchant/customer/create-order.md [hunk 1]
```diff
@@ -3,6 +3,8 @@
 To create a customer order post an [Order Creatable](../order/reference.html#order) to the order endpoint, 
 the `customer` field must be set to a customer id and the `payment` field must be a [Customer Payment Creatable](./reference.html#customer-payment).
 
+An order can also be with a specific customer method by [Selecting Customer Method](./payment-methods.html#select-customer-method).
```
Comment (Magnus93) [inline docs/merchant/customer/create-order.md:6]:
```suggestion
An order can also be created with a specific customer method by [Selecting Customer Method](./payment-methods.html#select-customer-method).
```

File: docs/merchant/customer/payment-methods.md [hunk 2]
```diff
@@ -17,6 +17,23 @@ Authentication: Bearer <public.api.key> | Bearer <private.api.key>
 }
 ```
 
+## Selecting Customer Method
+Use the function `Customer.Method.toPayment()` in the repository <a target="_blank" href="https://github.com/intergiro/model">Model</a> to convert a customer method to a [card payment](../order/reference.html#card-payment). Put the [card payment](../order/reference.html#card-payment) on the [Order Creatable](../order/reference.html#creatable) and POST to the [order endpoint](../order/create.html).
```
Comment (Magnus93) [inline docs/merchant/customer/payment-methods.md:21]:
```suggestion
Use the function `Customer.Method.toPayment()` from the repository <a target="_blank" href="https://github.com/intergiro/model">Model</a> to convert a customer method to a [card payment](../order/reference.html#card-payment). Put the [card payment](../order/reference.html#card-payment) on the [Order Creatable](../order/reference.html#creatable) and POST to the [order endpoint](../order/create.html).
```

File: docs/merchant/customer/payment-methods.md [hunk 3]
```diff
@@ -17,6 +17,23 @@ Authentication: Bearer <public.api.key> | Bearer <private.api.key>
...
+		})
```
Comment (Magnus93) [inline docs/merchant/customer/payment-methods.md:34]:
```suggestion
})
```

File: docs/merchant/customer/payment-methods.md (hunk 2 above)
Comment (Magnus93) [reply docs/merchant/customer/payment-methods.md:21]: Also the maybe the link should go to npm 
https://www.npmjs.com/package/@payfunc/model

Comment (Magnus93) [inline docs/merchant/customer/payment-methods.md:21]:
```suggestion
To select a customer method, use the function `Customer.Method.toPayment()` from the repository <a target="_blank" href="https://github.com/intergiro/model">Model</a> to convert a customer method to a [card payment](../order/reference.html#card-payment). Then, put the [card payment](../order/reference.html#card-payment) on the [Order Creatable](../order/reference.html#creatable) and POST to the [order endpoint](../order/create.html).
```

Comment (Magnus93) [reply docs/merchant/customer/payment-methods.md:21]: Also the maybe the **Model** link should go to npm
https://www.npmjs.com/package/@payfunc/model

**Review by Magnus93 — CHANGES_REQUESTED** [review]

File: docs/merchant/customer/payment-methods.md [hunk 4]
```diff
@@ -17,6 +17,23 @@ Authentication: Bearer <public.api.key> | Bearer <private.api.key>
 }
 ```
 
+## Selecting Customer Method
+To select a customer method, use the function `Customer.Method.toPayment()` from the repository <a target="_blank" href="https://github.com/intergiro/model">Model</a> to convert a customer method to a [card payment](../order/reference.html#card-payment). Then, put the [card payment](../order/reference.html#card-payment) on the [Order Creatable](../order/reference.html#creatable) and POST to the [order endpoint](../order/create.html).
```
Comment (Magnus93) [inline docs/merchant/customer/payment-methods.md:21]: Maybe use the npm link to **model** instead?
https://www.npmjs.com/package/@payfunc/model

**Review by Magnus93 — CHANGES_REQUESTED** [review]

**Review by nilssonemma — APPROVED** [review]

**Review by fredrikbryntesson — APPROVED** [review]

**Pull request merged**